docs/integrations/istio.md

```diff
@@ -0,0 +1,334 @@
+---
+weight: 1
+title: Istio
+menu:
+  docs:
+    parent: operator-integrations
+    weight: 3
+---
+
+
+In this guide, we'll walk through deploying and using VictoriaMetrics with [Istio](https://istio.io/) [service mesh](https://en.wikipedia.org/wiki/Service_mesh).
+Istio is a tool that helps manage network traffic between your applications in Kubernetes.
+We'll start with a simple setup and gradually add more advanced configurations,
+covering common challenges and best practices along the way.
+We recommend stop with the minimal viable configuration that meets your needs. 
+There is no need to add unnecessary complexity.
+
+## Prerequisites
+
+Before we dive into the details, let's set up a test Kubernetes cluster with VictoriaMetrics and Istio.
+This setup will be the foundation for the rest of the tutorial, so make sure to follow each step carefully.
+We'll install Istio first without enabling the mesh network. Later, we'll enable it in permissive mode and then strict mode,
+showing how to adapt VictoriaMetrics to work with each setting.
+Than, we proceed by installing a basic VictoriaMetrics setup, as outlined in the [Quick Start](https://docs.victoriametrics.com/operator/quick-start/) documentation.
+
+Create a test Kubernetes cluster using [Kind](https://kind.sigs.k8s.io/):
+```sh
+kind create cluster --name=istio;
+
+# Output:
+# ...
+# Set kubectl context to "kind-istio"
+# You can now use your cluster with:
+#
+# kubectl cluster-info --context kind-istio
+```
+
+Install Istio:
+```sh
+curl -L https://istio.io/downloadIstio | sh -;
+./istio-1.26.0/bin/istioctl install -f ./istio-1.26.0/samples/bookinfo/demo-profile-no-gateways.yaml -y;
```

Comment:
this pins the version and we'll likely forget about it in a year or two. Shall we now add a disclaimer that user has to check the version?

Reply:
Pinned version to 1.26.0